backend/src/components/permissionHelpers.js

```diff
@@ -1,55 +1,50 @@
 const log = require('npmlog');
-const utils = require('./utils');
 
+const { userService } = require('../services');
 
 const permissionHelpers = {
   // TODO: this is likely soon to be refactored out, as we will be pulling acronyms from the DB, not from access roles
```

Comment:
Still need this?

Reply:
The `filterAppAcronymRoles` function is still being left in as it's a part of the pathway where we load acronym information from JWT into the database. We elected at this time to leave that pathway in as it'd be an easy interim solution to populate the DB with new acronyms/user associations through Keycloak instead of having to do a direct DB query to insert. The TODO comment I left in because at some point further down the line when we remove this pathway we'll want to remove this function in its entirety as well.

Reply:
Right, gotcha